Profile/README.md Page for Meshery

nwanduka opened the pull request:
**Notes for Reviewers**

This PR fixes #8680

Here's the figma design:
![Meshery Profile](https://github.com/meshery/meshery/assets/122698422/e2ce613c-b46c-457e-84b4-4ac19bc6721d)
https://www.figma.com/file/vXAzeVtZhbCQrr9rRrpAUJ/Designs-For-Layer5?type=design&node-id=375-164&mode=design

I'd appreciate your reviews on this. @leecalcote 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 

<!--
Thank you for contributing to Meshery! 

Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

leecalcote reviewed (changes requested):
Service mesh management isn't Meshery's focus and should not be highlighted as it is just aa small part of Meshery does.

leecalcote commented on .github/profile/README.md:
```diff
@@ -0,0 +1,44 @@
+# Welcome to Meshery - Your Multi-Mesh Manager!
+
+Meshery is a self-service engineering platform that enables collaborative design and operation of cloud native infrastructure. Whether you're a seasoned DevOps pro, a curious developer, or an infrastructure enthusiast, Meshery empowers you to seamlessly design, deploy, and operate your cloud-native systems with ease.
+
```
"seasoned DevOps pro, a curious developer, or an infrastructure enthusiast," will need to be changed

nwanduka replied:
I removed it entirely. How's that?

leecalcote commented:
No more service mesh / multi-mesh, @nwanduka

nwanduka commented:
I've made the requested edits @leecalcote. Please take a look at it when you can.

abhijeetgauravm commented:
Hey @nwanduka  Let's discuss this on Meshery Development Meeting tomorrow at 8:30 PM IST / 9 AM Central time. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#).

leecalcote commented:
Apologies, I was out last week at KubeCon.

leecalcote commented:
Reminder, @Victoria Nduka: more more service mesh.

nwanduka commented:
> Apologies, I was out last week at KubeCon.

Welcome back

nwanduka commented:
> Reminder, @victoria Nduka: no more service mesh.

I've corrected this. Replaced "service mesh" with "cloud native technology" and other appropriate text. Please check it out when you can.

PS: I think you tagged the wrong Victoria 🧐

leecalcote commented:
@nwanduka you're closing in. There are a couple of things that need to be done:

1) this profile will go in a new repo: the .github repo (yet to be created).
2) please attempt to reuse similar, rich styling of things like the boxes on the page. Look to the other designs in the design for our examples, if you would (e.g. https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?type=design&node-id=21084-6782&mode=design&t=QiuJk1320dkL6LO4-4)
3) please move your designs into the community design file so that they can be maintained over time, people can access and review/comment.
